rpc/handlers.go

```diff
@@ -676,7 +676,17 @@ func (h *Handler) Events(args *EventsArg) (*EventsChunk, *jsonrpc.Error) {
 		}
 	}
 
-	if err = setEventFilterRange(filter, args.EventFilter.FromBlock, args.EventFilter.ToBlock, height); err != nil {
+	fromBlock := args.EventFilter.FromBlock
+	if fromBlock == nil {
+		fromBlock = &BlockID{Number: 0}
+	}
+
+	toBlock := args.EventFilter.ToBlock
+	if toBlock == nil {
+		toBlock = &BlockID{Latest: true}
+	}
```

Comment:
we can move these to blockchain.EventFilter

Reply:
Or maybe `setEventFilterRange`? `id.Latest` panics if it's `nil`

Reply:
The reason I think the default behaviour should be defined in the EventFilter object instead of the RPC is that if the EventFilter is ever used outside of the RPC, it should behave reasonably with just the default values

Reply:
we can make setEventFilterRange a noop if block id is nil